javascript/javascript3/week2/yesNo.js

```diff
@@ -0,0 +1,180 @@
+const movies = fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json')
+.then(response=>response.json())
+   .then(data => data.map(badmovies => badmovies).filter(movieList => movieList.rating < 6 && movieList.year >= 2000));
+   console.log(movies);
+
+ // async method 
+
+ async function asyncAwait(){
+   try{
+     const data = await fetch('https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json');
+     const movie = await data.json()
+     const badMovies = await movie.map(badMovie=>badMovie).filter(badMovieList=>
```

Comment:
I noticed you're using the `.map` function which is a very useful function for transforming list items. However for this task we don't need to be transforming the items, and as you can see we're just mapping the items to be the same, so in essence we're just iterating through the list without doing anything to it. So we can safely remove the call to `.map` :)

An example of where we can use `map` is if we wanted to create an array that contains the years of every movie. So we an do this: `const moviesYears = movies.map(movie => movie.year)`.